CONTRIBUTING.md

```diff
@@ -44,3 +44,4 @@ Here is a sample _good_ Git commit log message:
 
 Finally, a contribution is not a good contribution unless it comes with unit tests, integration tests and
```

Comment:
Would it make sense to add test cases for the LightStateConverter to make sure that it works as expected?